Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3570,6 +3571,72 @@ def fetch_items(command="fetch-incidents"):
     return last_run, items
 
 
+def get_cnapp_assets():
+    last_run = demisto.getAssetsLastRun()
+    demisto.debug(f"Starting a new cnapp fetch assets execution with {last_run=}")
+    snapshot_id = last_run.get("snapshot_id", str(round(time.time() * 1000)))
+    offset = int(last_run.get("offset", 0))
+    total_fetched_until_now = int(last_run.get("total_fetched_until_now", 0))
+    new_last_run = {}
+    limit = 100
+    endpoint = "/container-security/combined/container-alerts/v1"
+    params = {"offset": offset, "limit": limit}
+
+    demisto.info(f"Calling {endpoint=} with {params=}")
+    response = http_request("GET", endpoint, params)
+
+    cnapp_alerts = response.get("resources", [])
+    total_detections = demisto.get(response, "meta.pagination.total")
```

Comment:
what happens if a detection gets solved in the middle of a collection? Will we run forever/paginate out?

Reply:
then the total detections obtained from the meta.pagination.total should be updated.